pkg/controllers/customcluster_helper.go

```diff
@@ -125,6 +125,8 @@ type ConfigTemplateContent struct {
 	PodCIDR     string
 	// CNIType is the CNI plugin of the cluster on VMs. The default plugin is calico and can be ["calico", "cilium", "canal", "flannel"]
 	CNIType string
+	// VIPConfig is the config of VIP for HA when the EnableVIP is set to "true".
+	VIPConfig string
```

Comment:
can we structure this this, and render to string when marshalling?

Reply:
+1